normandy/base/middleware.py

```diff
@@ -117,3 +120,25 @@ def process_request(self, request):
                 },
             )
         return response
+
+
+def response_metrics_middleware(get_response):
+    def middleware(request):
+        start_time = time.time()
+        response = get_response(request)
+        delta = time.time() - start_time
+
+        view = request.resolver_match.func
+        view_name = f"{view.__module__}.{view.__name__}"
+        metrics.timing(
+            "response",
+            value=delta * 1000.0,
```

Comment:
Why a float? I thought markus wanted an int.

Reply:
This code is mostly [copied from Markus](https://github.com/willkg/markus/blob/master/markus/main.py#L344-L347), so I think it's fine. I've also tested this a bit with the debug stats logger, and it seems to work fine.